packages/remix-cloudflare-pages/worker.ts

```diff
@@ -1,17 +1,30 @@
-import type { ServerBuild, AppLoadContext } from "@remix-run/server-runtime";
-import { createRequestHandler as createRemixRequestHandler } from "@remix-run/server-runtime";
+import type { AppLoadContext, ServerBuild } from "@remix-run/cloudflare";
+import { createRequestHandler as createRemixRequestHandler } from "@remix-run/cloudflare";
+
+/**
+ * A function that returns the value to use as `context` in route `loader` and
+ * `action` functions.
+ *
+ * You can think of this as an escape hatch that allows you to pass
+ * environment/platform-specific values through to your loader/action.
+ */
+export type GetLoadContextFunction<Env = any> = (
+  context: EventContext<Env, any, any>
```

Comment:
Why does `EventContext` need to be parameterized for CF pages but [not for CF workers](https://github.com/remix-run/remix/blob/main/packages/remix-cloudflare-workers/worker.ts#L18)?

Reply:
It was already parameterized before, so I just kept it like this.
```ts
getLoadContext?: (context: EventContext<Env, any, any>) => AppLoadContext;
```

Happy to create another PR to do it for `remix-cloudflare-workers` as well if you want me to.